data/events/scripts/player.lua

```diff
@@ -347,19 +353,20 @@ function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder,
 		return false
 	end
 
-	-- Players cannot throw items on reward chest
-	local tileChest = Tile(toPosition)
-	if tileChest and tileChest:getItemById(ITEM_REWARD_CHEST) then
-		self:sendCancelMessage(RETURNVALUE_NOTPOSSIBLE)
-		self:getPosition():sendMagicEffect(CONST_ME_POFF)
-		return false
-	end
+	if tile then
```

Comment:
where is the variable tile?

Reply:
https://github.com/opentibiabr/canary/blob/ea86060977bfbb77af0670b399c75e22563aa8e3/data/events/scripts/player.lua#L248